packages/breadcrumbs/src/index.js

```diff
@@ -1 +1 @@
-export { default } from './Breadcrumbs';
+export { default } from './Breadcrumbs.js';
```

Comment:
Is there a specific reason for adding file extensions? 
We do not tend to that and now it's a mix 😕

Reply:
No you're right, its not needed. It's because I initially renamed the file to be a jsx file, but that bugged the whole thing so ended up reverting.